### ISSUE TITLE
Pinpad callback

### ISSUE THREAD

**Issue by axic** [trezor/trezor.js#21]
In the [example](https://github.com/trezor/trezor.js/blob/master/doc/example-list.js#L91-L95) the pinpad callback is defined as:

```js
function pinCallback(type, callback) {
    // We should ask the user for PIN and send it back as '1234', where 1 is the
    // top left position, 2 is the top middle position, etc.
    throw new Error('Nothing defined');
}
```

However while implementing an app, all my PINs were rejected. Looking at the mytrezor.com PIN popup it seems like that the layout is:
```
789
456
123
```

Doing the callback based on that works. Is only the example wrong or something else is going on? 

(I have also opensourced a React component for this: https://github.com/axic/trezor-react-pinpad)

**Comment by prusnak** [comment]
Example is unfortunately wrong. I fixed it in 0099057bcb356e9b774063748835f00fd080b16a

Python documentation is correct for example: https://github.com/trezor/python-trezor#pin-entering

**Comment by axic** [comment]
Thanks :+1:
